p4-fusion/commands/print_result.cc

```diff
@@ -6,15 +6,22 @@
  */
 #include "print_result.h"
 
+#include <utility>
+
+PrintResult::PrintResult(std::function<void()> stat, std::function<void(const char*, int)> out)
+    : onStat(std::move(stat))
+    , onOutput(std::move(out))
```

Comment:
rename these variables to something like

`onNextFile` and `onFileContentChunk` or something like that, plus some comment on when they're called.